app/advancedfilters/filterButtons.js

```diff
@@ -1,8 +1,7 @@
+'use strict';
 app.directive('filterButtons', function() {
   return {
     restrict: 'E',
-    template: '<button class="btn quickgo-btn" type="button" ng-click="reset()">Reset</button>      <button class="btn quickgo-btn" type="button" ng-click="apply()">Apply</button>',
-    link: function($scope, element, attrs) {
-    }
-  }
-})
+    template: '<button class="btn quickgo-btn" type="button" ng-click="reset()">Reset</button>      <button class="btn quickgo-btn" type="button" disabled="{{}}" ng-click="apply()">Apply</button>'
```

Comment:
Is there anything missing in the empty binding disabled="{{}}"?
This might be related to the "Apply" button being always disabled. Please check.